libraries/AP_GPS/AP_GPS.h

```diff
@@ -310,7 +310,8 @@ class AP_GPS
     }
 
     // the expected lag (in seconds) in the position and velocity readings from the gps
-    float get_lag() const { return 0.2f; }
+    float get_lag(uint8_t instance) const;
```

Comment:
I will have a look at it, but it would involve making changes in multiple files for no benefit other than a few bytes of storage.